charts/astronomer/templates/houston/cronjobs/houston-update-root-admin-password.yaml

```diff
@@ -0,0 +1,66 @@
+#######################################
+## Create Root Admin Job
+#######################################
+apiVersion: {{ include "apiVersion.batch.cronjob" . }}
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-update-root-admin-password-cronjob
+  labels:
+    tier: astronomer
+    component: houston-update-root-admin-password-cronjob
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  schedule: "0 5 31 2 0"
```

Comment:
This is likely not doing what you want. <https://crontab.guru/#0_5_31_2_0>

Reply:
```suggestion
  schedule: "0 5 31 2 *"   # https://crontab.guru/#0_5_31_2_*  At 05:00 on day-of-month 31 in February.
```

Reply:
tbh either what I have or what you have works for me. Both are fake cron schedules for non existant days which is what I want. You'll notice the suspend: true param. Which essentially prevents this from scheduling.

Reply:
Your original one will run every Sunday in Feb.

Reply:
ah yeah I was misreading cron guru thanks